server/ai_process.go

```diff
@@ -1013,41 +1013,25 @@ func submitLiveVideoToVideo(ctx context.Context, params aiRequestParams, sess *A
 		}
 		return nil, err
 	}
+	// Send request to orchestrator
 	resp, err := client.GenLiveVideoToVideoWithResponse(ctx, req)
 	if err != nil {
 		return nil, err
 	}
 	if resp.JSON200 != nil {
 		// append orch hostname to the given url if necessary
-		appendHostname := func(urlPath string) (*url.URL, error) {
-			if urlPath == "" {
-				return nil, fmt.Errorf("invalid url from orch")
-			}
-			pu, err := url.Parse(urlPath)
-			if err != nil {
-				return nil, err
-			}
-			if pu.Hostname() != "" {
-				// url has a hostname already so use it
-				return pu, nil
-			}
-			// no hostname, so append one
-			u := sess.Transcoder() + urlPath
-			return url.Parse(u)
-		}
-		pub, err := appendHostname(resp.JSON200.PublishUrl)
+		pub, err := url.Parse(resp.JSON200.PublishUrl)
```

Comment:
Should preserve the old behavior here of appending the hostname if necessary, because I am not sure if orchestrators are guaranteed to have a ServiceURI set

Reply:
correcting myself here - looks like it _is_ set but still not sure we should rely on it instead of a relative path from  [this comment](https://github.com/livepeer/go-livepeer/pull/3263#discussion_r1853282905)

Reply:
Agree, I reverted back to `sess.Transcoder()` from the gateway, see above comment